backend/defence.js

```diff
@@ -38,9 +42,37 @@ function isDefenceActive(id) {
   return false;
 }
 
+function generate_random_string(string_length){
+  let random_string = '';
+  let random_ascii;
+  for(let i = 0; i < string_length; i++) {
+      random_ascii = Math.floor((Math.random() * 25) + 97);
+      random_string += String.fromCharCode(random_ascii)
+  }
```

Comment:
Could move the declaration of `random_ascii` into the for loop and make it `const`.